src/types/range.ts

```diff
@@ -18,6 +18,12 @@ export interface Range extends Cloneable<Range> {
   readonly rangeData: RangeData;
 }
 
+// TODO rename this
+export interface SmallRange {
+  sheetId: UID;
+  zone: Zone;
+}
```

Comment:
Please help me.
I originally thought it would be great to be able to assign a `Range` to this `SmallRange` (that's the idea of having the two properties "sheetId" and "zone" properties). It's kinda a range but it lacks the (possible) unboundedness.

Should it be more like a "SheetyZone" such as ` { sheetId, left, right, top, bottom }` ?

Reply:
I would go for the idea to reduce the scope. If it's not use elsewhere for now, let's keep the name simple and clear. If necessary, it will still be possible to rename it later.